pkg/logs/input/docker/scanner.go

```diff
@@ -144,21 +144,25 @@ func (s *Scanner) setup() error {
 	}
 
 	// Start tailing monitored containers
-	s.scan(false)
+	s.scan()
 	return nil
 }
 
 // setupTailer sets one tailer, making it tail from the beginning or the end,
 // returns true if the setup succeeded, false otherwise
-func (s *Scanner) setupTailer(cli *client.Client, container types.Container, source *config.LogSource, tailFromBeginning bool, outputChan chan message.Message) bool {
+func (s *Scanner) setupTailer(cli *client.Client, container types.Container, source *config.LogSource, outputChan chan message.Message) bool {
 	log.Info("Detected container ", container.Image, " - ", s.humanReadableContainerID(container.ID))
-	t := NewTailer(cli, container.ID, source, outputChan)
-	err := t.recoverTailing(s.auditor, tailFromBeginning)
+	tailer := NewTailer(cli, container.ID, source, outputChan)
+	since, err := Since(s.seeker, container, tailer.Identifier())
+	if err != nil {
+		log.Warnf("Could not recover last committed offset for container %v: %v", container.ID[:12], err)
```

Comment:
I'd be wary of static `:12` a bit everywhere in the code which is very docker-tainted, we might stumble one day on a container runtime that has shorter ids. I think we can spare a short method somewhere that checks the length of the id before returning the shortened version.